src/store/modules/product/actions.ts

```diff
@@ -42,6 +42,22 @@ const actions: ActionTree<ProductState, RootState> = {
     return resp;
   },
 
+  async fetchProduct ({ commit }, { productId }) {
+    const resp = await ProductService.fetchProducts({
+      "filters": ['productId: ' + productId ]
+    })
+    if (resp.status === 200 && !hasError(resp)) {
+      const product = resp.data.response.docs[0];
+      if (resp.data) commit(types.PRODUCT_ADD_TO_CACHED, product);
+    }
+    // TODO Handle specific error
+    return resp;
+  },
+
+  async updateCurrent({ commit }, { product }) {
```

Comment:
Why not just `product` instead of `{ product }` ?

Reply:
Updated